modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/ui/Common.kt

```diff
@@ -82,8 +84,12 @@ internal fun ShareStoryButton(
 ) {
     val scope = rememberCoroutineScope()
 
-    OutlinedEoyButton(
-        text = stringResource(LR.string.end_of_year_share_story),
+    CircleIconButton(
+        size = 48.dp,
```

Comment:
The design has the touch target at 34.dp but it feels very small on Android so I might ask the designers to try it at 48.dp first. It also matches the Material spec. https://m3.material.io/foundations/designing/structure#b421e522-35a6-47cd-bfc6-bf94cbacf7bb

<img width="109" height="134" alt="Image" src="https://github.com/user-attachments/assets/8141bb4e-50d2-4b3f-9400-21f7fdd11adf" />